src/playground/App.js

```diff
@@ -125,12 +125,33 @@ const App = () => {
         setOptions(newOptions);
         storeState({ newOptions });
     };
+    const [showConfigMenu, toggleConfigMenu] = useState(false);
+    const [isConfigHidden, toggleConfig] = useState(window.matchMedia("(min-width: 1023px)").matches);
+
+    useEffect(() => {
+        const mq = window.matchMedia("(min-width: 1023px)");
+
+        const ConfigToggler = () => {
+            toggleConfig(mq.matches);
+        };
+
+        mq.addEventListener("change", ConfigToggler);
+
+
+        return () => mq.removeEventListener("change", ConfigToggler);
+    }, []);
+
 
     return (
         <div className="playground-wrapper">
             <div className="playground__config-and-footer">
                 <section className="playground__config" aria-labelledby="playground__config-toggle">
-                    <button className="playground__config-toggle" id="playground__config-toggle" hidden>
+                    <button className="playground__config-toggle" id="playground__config-toggle" onClick={ () => {
+                        toggleConfigMenu(value => !value);
```

Comment:
```suggestion
                        setShowConfigMenu(!showConfigMenu);
```

Reply:
I'd always prefer an update function instead of the value itself as we are updating the previous value.